server/internal/testutil/postgres.go

```diff
@@ -31,8 +32,13 @@ func WithPgDB() IntegrationDependency {
 	return IntegrationDependency(func(ctx context.Context, t *testing.T) (context.Context, func()) {
 		t.Helper()
 
+		img := os.Getenv("TESTCONTAINER_POSTGRES_IMAGE")
```

Comment:
❓ Is there a use case for this? 🤔

Reply:
I just don't like hardcoding magic values, it's not really necessary but I wanted to have an escape hatch